Meta/lint-ports.py

```diff
@@ -347,6 +347,10 @@ def run():
     from_table_set = set(from_table.keys())
     ports_set = set(ports.keys())
 
+    if list(from_table.keys()) != sorted(from_table.keys()): # Dictionaries are sorted by the order of addition, converting to a set loses this property
```

Comment:
This comment might as well be above the `if` instead of behind it, that also solves the line length CI issue.